Breadcrumbs.py

```diff
@@ -56,7 +56,7 @@ def make_breadcrumbs(view, current_row, shorten=False):
 
   settings = sublime.load_settings('Breadcrumbs.sublime-settings')
   tab_size = get_tab_size(view)
-  default_breadcrumb_regex = settings.get('breadcrumb_regex', u'^\s*(?P<name>.*\\S)')
+  default_breadcrumb_regex = settings.get('breadcrumb_regex', u'')
```

Comment:
There isn't actually a reason for this fallback/default, since it's already set in the package default settings. These settings not loading will either point to a serious problem elsewhere, or someone manually an wilfully editing the default settings (which if installed via package control also entails unpacking and moving the package). Same goes for the other cases where settings are read from the settings file: I don't think there is a need for a default. In fact, the default only introduces duplication.